### PR TITLE
Add missing parameter to command built by batchrun for OR-Tools

### PR DESCRIPTION
**Pull request by flomnes**

### CODE REVIEW

File: src/tools/batchrun/main.cpp [hunk 1]
```diff
@@ -293,7 +293,7 @@ int main(int argc, char* argv[])
             if (optForceParallel)
                 cmd << " --force-parallel=" << *optForceParallel;
 	    if (ortoolsUsed)
- 	        cmd << " --ortools-solver=" << ortoolsSolver;
+ 	        cmd << " --use-ortools --ortools-solver=" << ortoolsSolver;
```
Comment (sylvlecl) [inline src/tools/batchrun/main.cpp]: We should check that ortools solver is not empty, otherwise the command will not be valid

**Review by sylvlecl — CHANGES_REQUESTED** [review]

File: src/tools/batchrun/main.cpp [hunk 2]
```diff
@@ -190,166 +189,179 @@ int main(int argc, char* argv[])
...
-            logs.fatal() << "The solver has not been found. specify --solver=" << solver;
-            return EXIT_FAILURE;
+            const auto availableSolvers = getAvailableOrtoolsSolverName();
+            if (auto it
+                = std::find(availableSolvers.begin(), availableSolvers.end(), ortoolsSolver);
+                it == availableSolvers.end())
+            {
+                logs.error() << "Please specify a solver using --ortools-solver. Available solvers "
+                             << availableOrToolsSolversString() << "";
+                return EXIT_FAILURE;
```
Comment (flomnes) [inline src/tools/batchrun/main.cpp:202]: Change 1

File: src/tools/batchrun/main.cpp [hunk 3]
```diff
@@ -190,166 +189,179 @@ int main(int argc, char* argv[])
...
+                if (optNoTSImport)
+                    cmd << " --no-ts-import";
+                if (optIgnoreAllConstraints)
+                    cmd << " --no-constraints";
+                if (optParallel)
+                    cmd << " --parallel";
+                if (optForceParallel)
+                    cmd << " --force-parallel=" << *optForceParallel;
+                if (ortoolsUsed)
+                    cmd << " --use-ortools --ortools-solver=" << ortoolsSolver;
```
Comment (flomnes) [inline src/tools/batchrun/main.cpp:307]: Change 2

**Review by flomnes — COMMENTED** [review]
Most changes are due to auto-formatting. I highlighted "real" changes.

**Review by sylvlecl — APPROVED** [review]

**Pull request merged**